src/main/resources/application-localstack.yml

```diff
@@ -54,10 +54,10 @@ hmpps.sqs:
     eventincidents:
       queueName: ${random.uuid}
       dlqName: ${random.uuid}
+      errorVisibilityTimeout: 0
```

Comment:
Too much context switching - added against csip instead of incidents (now corrected)